app-text/mupdf/mupdf-1.21.1-r1.ebuild

```
@@ -157,7 +157,8 @@ src_install() {
 	fi
 
 	# Respect libdir (bug #734898)
-	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || die "Failed to sed pkgconfig file to respect libdir in src_install()"
+	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc \
+		|| die "Failed to sed pkgconfig file to respect libdir in src_install()"
```

I often write it like:
```suggestion
	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || 
		die "Failed to sed pkgconfig file to respect libdir in src_install()"
```
No need for backlash either then.

---

Ah, I didn't know that. Thanks!